sql/src/main/java/io/crate/planner/consumer/DistributedGroupByConsumer.java

```diff
@@ -190,6 +190,13 @@ public AnalyzedRelation visitQueriedTable(QueriedTable table, Context context) {
         }
 
         @Override
+        public AnalyzedRelation visitInsertFromQuery(InsertFromSubQueryAnalyzedStatement insertFromSubQueryAnalyzedStatement, Context context) {
+            new InsertFromSubQueryConsumer.InnerRelationPlanner<Context, AnalyzedRelation>().process(insertFromSubQueryAnalyzedStatement,
```

Comment:
can the InnerRelationPlanner be changed to a singleton and then expose a static method on the InsertFromQueryConsumer?